examples/copy-publish.js

```diff
@@ -0,0 +1,43 @@
+// To run this script, use `node share.js`
```

Comment:
Tiny nitpick, but can you either update this to use the name of this script or remove this line?

Reply:
Updated and also changed some of the variable names which I noticed freaked out a less intelligent text editor I opened the file in (public is a keyword in typescript, though I think during runtime in JS at least it works perfectly fine).